src/components/Main/Greetings.js

```diff
@@ -0,0 +1,20 @@
+const language = ['de', 'en', 'es', 'fr'];
```

Comment:
Buenisismo la destructuracion que elegancia obtener el indice y sacar el valor del segundo arreglo 🔥 
```suggestion
const language = ['de', 'en', 'es', 'fr'];
let hello = ['Hallo', 'Hi', 'Arre', 'Bonjour'];

const Greetings = ({ lang, children }) => {
let  findLang = language.indexOf(lang);
let  word = hello[findLang];
  return (
    <>
      <div style={{ borderStyle: 'solid', margin: 5, textAlign: 'left' }}>
        <p>
          {word}, {children}
        </p>
      </div>
    </>
  );
};

export default Greetings;
```